app/assets/v2/js/pages/bounty_details.js

```diff
@@ -65,6 +65,13 @@ const token_value_to_display = function(val) {
   return Math.round((parseInt(val) / Math.pow(10, document.decimals)) * 1000) / 1000;
 };
 
+const case_insensitive_cmp = (val1, val2) => {
```

Comment:
this could probably live in shared.js...

Reply:
I agree with that idea!